point-of-sale/src/pages/new.tsx

```diff
@@ -1 +1,12 @@
 export { default } from '../client/components/pages/NewPage';
+
+export function getServerSideProps() {
```

Comment:
Hmm, do you think it's better to export these here? My inclination is to place them with the component, but I don't have a strong opinion (or know if it makes a difference).

Reply:
Agreed, didn't realise exporting from the component would work. Updated to `export { default , getServerSideProps } from '../client/components/pages/NewPage';` which sorts it and I think that's the better pattern